cpu_features.h

```diff
@@ -23,6 +23,7 @@
 extern void cpu_check_features();
 
 /* adler32 */
+typedef uint32_t (*adler32_func)(uint32_t adler, const unsigned char *buf, size_t len);
```

Comment:
Nitpicking here, but wouldn't it be better to have an empty line between the typedef and the big block of externs, just to set those a little apart to make it easier to read?

Reply:
Fixed.